### ISSUE TITLE
Clarify scope of project

### ISSUE THREAD

**Issue by juliohm** [unfoldtoolbox/UnfoldMakie.jl#122]
Can you please comment on the scope of this project? I see that the subtitle includes "EEG" in it, does it mean that the plotting functions provided here are tied to this type of data?

Some plots like parallel coordinates are general and can be used with any table, not just EEG data.

**Comment by behinger** [comment]
hey, most plotting functions here have direct relations to EEG, or at least a relation to the Unfold.jl Toolbox. 

I agree parallel coordinates are much more general, I'm thinking of pushing them as a core plotting tool to Makie.jl - or maybe something like we did for TopoPlots.jl but it is much more work to be done to make it good.

If you are interested in parallel coordinates, have a look here: https://github.com/unfoldtoolbox/UnfoldMakie.jl/pull/121 where we right now rewrote the whole thing :)

**Comment by juliohm** [comment]
Yes! It would be nice to have this ParallelCoordinatePlots.jl package as a standalone package for Tables.jl tables.

We are hosting some of these specialized plots directly in MakieOrg. For example, the Biplots.jl was a plot I contributed that is independent of the Makie.jl release cycle: https://github.com/MakieOrg/Biplots.jl

**Comment by juliohm** [comment]
Any update regarding this issue?

**Comment by behinger** [comment]
I tried at some point to implement it via as a new `Block`- but at that point it was not super easy and I gave up after a while.

Now with the new `ComputeGraph` thingy it might be easier to do

**Comment by juliohm** [comment]
The `ComputeGraph` gives you a method to update plot attributes without sync issues. I wonder how it could help with the split here. What is the challenge exactly?
